Sources/SwiftTemplate.swift

```diff
@@ -3,9 +3,16 @@
 // Copyright (c) 2017 SwiftGen
 // MIT Licence
 //
-
+import Foundation
 import Stencil
 
+#if os(Linux)
+#if swift(>=3.1)
```

Comment:
only because I was playing around and wanted to let him compare. I'll delete mine and push the canges here, gimme a moment while I rebase the rest of the branches (so many branches :P )